packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -238,13 +248,13 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
       this.context.updateClientState(clientState);
     };
 
-    this.handlers = createDefaultCallingHandlers(callClient, callAgent, deviceManager, undefined);
-
     this.onClientStateChange = onStateChange;
 
     this.subscribeDeviceManagerEvents();
 
     this.callClient.onStateChange(onStateChange);
+
+    this.handlers = createHandlers(callClient, callAgent, deviceManager, undefined);
```

Comment:
did this move for a specific reason (i.e. this now _must_ be called after over functions)? If so add a comment that this must be called at the end

Reply:
I don't think this moved for any specific reason.

Reply:
Previously it was because conditional compilation, but we extract the logic to createHandlers in the end, so I think it's good to move it back to the place